meson.build

```diff
@@ -1,4 +1,4 @@
-project('snapd-desktop-integration', 'c', version: '0.9')
+project('snapd-desktop-integration', 'c', version: '0.10')
```

Comment:
I'd just leave the version number as 0.9 and bump it when making the release.

Reply:
Done.